src/Compiler/Checking/CheckDeclarations.fs

```diff
@@ -4418,6 +4418,33 @@ module TcDeclarations =
             let isAtOriginalTyconDefn = true
             let core = MutRecDefnsPhase1DataForTycon(synTyconInfo, SynTypeDefnSimpleRepr.Exception r, implements1, false, false, isAtOriginalTyconDefn)
             core, extra_vals_Inherits_Abstractslots @ extraMembers
+            
```

Comment:
Thanks for the review. I'm looking at the [IWSAM Implementation](https://github.com/dotnet/fsharp/pull/13119/files) to see if there is a better place to do this where I have access to the implemented members and the interface definition. Maybe @vzarytovskii knows the right place?

Reply:
This is the place where "no implementation was given for" messages are being generated:

https://github.com/dotnet/fsharp/blob/72ee6e448e0c86865915580abe37df363b980f45/src/Compiler/Checking/MethodOverrides.fs#L318

it would feel like the right place, to have the logic there.

Sorry that I initially pointed you in the chat at CheckDeclaration as entry point where the slots start to be processed.

Reply:
Moved the check to a more appropriate place now